src/formatters/junit.js

```diff
@@ -0,0 +1,42 @@
+/*eslint no-console: "off"*/
+
+function printResults(results) {
+  const testCases = results.map(result => {
+    const file = result.filePath;
+    return {
+      _attributes: {
+        name : file
+      },
+      error : result.errors.map(error => {
+        return {
+          _attributes: {
+            message: `${error.message}`,
```

Comment:
No need for the template literal as you are not combining a string with a variable. `message: error.message` should surfice.